2.6.md

```diff
@@ -692,6 +713,11 @@ There are also several subordinate objects that provide detailed data to potenti
     <td>object; recommended</td>
     <td>Details via an <code>App</code> object (Section 3.2.14) about the publisher’s app (i.e., non-browser applications). Only applicable and recommended for apps.</td>
   </tr>
+  <tr>
+    <td><code>dooh</code></td>
+    <td>object</td>
+    <td>This object should be included if the ad supported content is a Digital Out-Of-Home screen. A bid request with a DOOH object must not contain a site or app object.</td>
```

Comment:
This is a bit of a nitpick, so feel free to ignore. However, since this is mutually exclusive with `site` and `app`, we can make the descriptions look the same, e.g.:

> Details via an <code>DOOH</code> object (Section X) about the publisher’s Digital Out-Of-Home screen(s). Only applicable and recommended for Digital Out-Of-Home.